cmd/update.go

```diff
@@ -88,8 +88,7 @@ func init() {
 
 	checkBindFlagError(viper.BindPFlag("update.username", updateCmd.Flags().Lookup("username")))
 	checkBindFlagError(viper.BindPFlag("update.password", updateCmd.Flags().Lookup("password")))
-	checkBindFlagError(viper.BindPFlag("update.transfer-protocol", updateCmd.Flags().Lookup("transfer-protocol")))
-	checkBindFlagError(viper.BindPFlag("update.protocol", updateCmd.Flags().Lookup("protocol")))
+	checkBindFlagError(viper.BindPFlag("update.scheme", updateCmd.Flags().Lookup("scheme")))
```

Comment:
I was going to ask about the difference between `protocol` and `transfer-protocol`, so having a single `scheme` option makes more sense.